### PR TITLE
add utter_image_url

### PR DESCRIPTION
**Pull request by rajithacharith**
**Proposed changes**:
- When trying to send image from custom actions there is no function in the dispatcher to send image as a URL. But in rasa core, it supports the image sending and only needs to send image url formated as {"image" : image_url }. Added utter_image_url function and implementation to CollectingDispatcher class. 

**Status (please check what you already did)**:
- [+] made PR ready for code review
- [- ] added some tests for the functionality
- [- ] updated the documentation
- [- ] updated the changelog

### CODE REVIEW

**Comment by akelad** [comment]
Thanks for submitting this PR, we'll give it a review as soon as possible

File: rasa_sdk/executor.py [hunk 1]
```diff
@@ -116,6 +116,14 @@ def utter_custom_json(self, json_message, **kwargs):
 
         self.messages.append(json_message)
 
+    def utter_image_url(self, image, **kwargs):
```
Comment (erohmensing) [inline rasa_sdk/executor.py]: Can you add types (as above) please?

File: rasa_sdk/executor.py [hunk 2]
```diff
@@ -116,6 +116,14 @@ def utter_custom_json(self, json_message, **kwargs):
...
+        """ sends image url to the output channel as image"""
```
Comment (erohmensing) [inline rasa_sdk/executor.py]:
```suggestion
        """Sends url of image attachment to the output channel."""
```

File: rasa_sdk/executor.py [hunk 3]
```diff
@@ -116,6 +116,14 @@ def utter_custom_json(self, json_message, **kwargs):
...
+
+        json_message = {"image": image}
```
Comment (erohmensing) [inline rasa_sdk/executor.py]: please change these `json_message` to `message` (`utter_custom_json` is a little special -- the rest are just `message`s.)

**Comment by rajithacharith** [comment]
Thanks for the quick reply :)
I updated the code as requested

**Pull request merged**